samples/drivers/spi_flash/src/main.c

```diff
@@ -14,11 +14,11 @@
 /* NB: W25Q16DV is a JEDEC spi-nor device, but has a separate driver. */
 #define FLASH_DEVICE CONFIG_SPI_FLASH_W25QXXDV_DRV_NAME
 #define FLASH_NAME "W25QXXDV"
-#elif (CONFIG_SPI_NOR - 0) || defined(DT_INST_0_JEDEC_SPI_NOR_LABEL)
-#define FLASH_DEVICE DT_INST_0_JEDEC_SPI_NOR_LABEL
+#elif (CONFIG_SPI_NOR - 0) || DT_NODE_HAS_PROP(DT_INST(0, jedec_spi_nor), label)
```

Comment:
Better handled as node existence checks?

Reply:
I agree but I believe these are automated conversions.  We can probably do a replacement as a separate cleanup after everything's completed.  I suspect there are more like this, or will be soon.

Reply:
I fixed this up,  There are a small handful of cases in which we test for label that I'll send some cleanup PRs for.